src/components/common/Tabs/Tabs.tsx

```diff
@@ -53,6 +53,10 @@ const Wrapper = styled.div`
   }
 `
 
+export const TabList = styled.div`
+  /* stylelint-disable no-empty-block */
```

Comment:
should we remove this comment?

Reply:
Looks like the whole component (TabList) could be removed

Reply:
@matextrem just remove this component and you are ready to merge